scripts/run-reset.ts

```diff
@@ -7,25 +7,25 @@ const logger = console;
 
 fs.writeFileSync('reset.log', '');
 
-// let results = [];
-const cleaningProcess = spawn('git', [
-  'clean',
-  '-xdf',
-  '-n',
-  '--exclude=".vscode"',
-  '--exclude=".idea"',
-]);
+const cleaningProcess = spawn('git', ['clean', '-xdf', '-n']);
 
 cleaningProcess.stdout.on('data', (data) => {
   if (data && data.toString()) {
-    const l = data
+    data
       .toString()
       .split(/\n/)
-      .forEach((i) => {
+      .forEach((i: string) => {
         const [, uri] = i.match(/Would remove (.*)$/) || [];
 
         if (uri) {
           if (
+            uri.match(/dist/) ||
+            uri.match(/.vscode/) ||
+            uri.match(/.idea/) ||
+            uri.match(/reset.log/)
+          ) {
+            // ignore, don't remove these files
```

Comment:
I think you can extract all these uri match assertions to variables and as a result you both have more readable code plus you can re-structure these conditionals to be simpler. This empty if is a bit weird